src/test/extensions/askUserForParam.spec.ts

```diff
@@ -198,7 +200,7 @@ describe("askUserForParam", () => {
       expect(res).to.equal("");
     });
   });
-  describe("askForParam", () => {
+  describe("askForParam with string param", () => {
```

Comment:
undo?

Reply:
This was to contrast with the [new test below "askForParam with secret param"](https://github.com/firebase/firebase-tools/pull/3901/files#diff-3e09552fce53dd0c656d89aaaea5adcffb1e8b33316dbfd5332afbf35fce1a9bR223).